src/editorApp/checkPage.js

```diff
@@ -160,6 +160,17 @@ export const init = () => {
 		debug( 'Gutenberg is not enabled.' );
 	}
 
+	const editor = wp?.data?.select?.( 'core/editor' );
+
+	// In the block editor, don't run the initial scan on auto-drafts.
+	if ( editor ) {
+		const postStatus = editor.getEditedPostAttribute( 'status' );
+		if ( ! postStatus || postStatus === 'auto-draft' ) {
+			return;
+		}
+	}
+
+	// Fallback for classic editor or for saved posts in the block editor.
```

Comment:
This comment implies there's a fallback for the classic editor but there's no fallback below. Can we add one or remove the comment?